### ISSUE TITLE
Export poll results (engagement) incorrect – Hide Results case

### ISSUE THREAD

**Issue by coopchange** [loomio/loomio#7475]
Hi @robguthrie could you help me sort this quickly? Would help me deliver a quick response/ solution to NZEI.

Isabella wants to know how many people have voted so far in her poll that has Hide Results Until Close turned on. Export poll is an obvious solution for her, but when I tested it, the engagement metrics were wrong; see the [export poll results](https://drive.google.com/file/d/1YRQxU8PFqUTxx2n067JfQKzofRHMBXDG/view?usp=sharing) and see the [poll from which](https://www.loomio.org/p/5yUJhbYl/test-hide-results) it was exported – I closed it in the meantime (to observe the discrepancy).

Reproduction
1. start poll with Hide Results…
2. user votes
3. poll creator exports results

Please and thanks!

**Comment by robguthrie** [comment]
What if you use the remind function?

**Comment by robguthrie** [comment]
Yep, it looks like you can see that if you click remind, then select
undecided voters, then click to expand that group of people.

**Comment by robguthrie** [comment]
And one probably should not be able to export a poll when hide results
until closed is on. I'd consider it a bug that it's available

**Comment by coopchange** [comment]
Cheers.

To your last comment: Yeah maybe, depends on the driving reason for the hide-results-until-close feature. The story I've heard a couple of times is "i want to prevent group think". So if admins can export results part way through, then it seems in line with this idea, and export poll seems a more straightforward way for an admin to see how a vote is going. I can also see your point of view though.

**Comment by coopchange** [comment]
Mm I'm going to have a check how the flow you propose would work for a group with hundreds of members... Isabella is keen to even just have a sense of the percentage of people who have engaged so far
^ that might be a useful product user story

**Comment by coopchange** [comment]
Ah okay I see that it says how many will be notified 👍

**Comment by coopchange** [comment]
FYI: Clicking to expand the group of people doesn't show you neither who has voted nor who has not (but not sure that that's a big deal for this case w Isabella).

Also had a wee bug where selecting "Everyone invited to vote" shows it will send 0 notifications where I can see that people have indeed been invited/ sent a notification for the poll. It flashed a number near to the correct one for a second before going to 0. Nothing notable in the console (just the regular 'vue... value type not supported' and socket.io...)

**Comment by coopchange** [comment]
FWIW: Response from NZEI:
> Further to this, there is no way for us to see how people have voted is there?

**Comment by robguthrie** [comment]
Remind will do that too, right?

I see the issue though. I can show this stuff on the poll page.

**Comment by robguthrie** [comment]
@coopchange I've fixed the main problem. The votes panel, which shows who has voted, will show now, on the poll page, when hide results until closed is active. It will just show who has voted and who has not voted.

I've also disabled export poll for these polls until you're officially able to see results.  - It's a completely undefendable position to allow admins to see results before the rest of the group. unless we tell everyone that's the case.
